front/tests/operational-studies.spec.ts

```diff
@@ -11,24 +11,43 @@ test.describe('Testing if all mandatory elements simulation configuration are lo
 
   test.beforeAll(async ({ browser }) => {
     const page = await browser.newPage();
+
     playwrightHomePage = new PlaywrightHomePage(page);
     await playwrightHomePage.goToHomePage();
+    await playwrightHomePage.page.route('**/projects/*', async (route) => {
```

Comment:
```suggestion
    await playwrightHomePage.page.route('**/projects/', async (route) => {
```
Can we remove this * if not needed ?

Reply:
(on the 3 new tests, project, scenario and study)

Reply:
It is needed, because those are search endpoints; the url to be catch have query string (eg: `projects/?q=blablabla`)